### DISCUSSION TITLE
Error when trying to run migrations on elastic beanstalk

### DISCUSSION THREAD

**Discussion by DavideCarvalho** [prisma/prisma#5882] [Q&A]
I'm trying to run ``npx prisma migrate deploy ==preview-feature`` on elastic beanstalk, but all I get is this error message down below.

Has anybody else tumbled on this problem?

![image](https://user-images.githubusercontent.com/15630886/109358241-34080100-7862-11eb-8e84-1bd101d3c2b6.png)

**Comment by ryands17** [comment]
@DavideCarvalho 👋 

Seems that EB cannot find `package.json`. Did you run `eb deploy` via the CLI? If so, could you check if all files are being uploaded properly along with the `prisma` folder.

Ideally I would suggest not running `migrate` commands in EB but before you deploy to EB either locally or automated via CI.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
